kubernetes/helmfile.yaml

```
@@ -10,6 +10,8 @@ repositories:
     url: https://helm.elastic.co
   - name: bitnami
     url: https://charts.bitnami.com/bitnami
+  - name: tempo
```

```suggestion
  - name: foo
```
Je préfère foo à tempo